draft-ietf-ppm-dap.md

```diff
@@ -1440,21 +1470,17 @@ struct {
 } InputShareAad;
 ~~~
 
-The Leader responds to well-formed requests with HTTP status code 201
```

Comment:
This nixes the specific prescription of 201 Created. See comment on the HPKE config section.